libs/ui/pages/auth/sign-up/organization/types.ts

```diff
@@ -0,0 +1,10 @@
+import { IDefaultForm, IForm } from '../../../../../utils/formik';
+
+export type OrganizationLogicType = {
+    handleOnSubmit(): void;
+    form: IForm<IOrganizationFormik> & IDefaultForm;
+};
+
+export interface IOrganizationFormik {
```

Comment:
```suggestion
export interface IOrganizationForm {
```

Later we will maybe not used formik anymore